test/AngularSpec.js

```diff
@@ -435,6 +435,95 @@ describe('angular', function() {
   });
 
 
+  describe('jq', function() {
+    var element;
+
+    beforeEach(function() {
+      element = document.createElement('html');
+    });
+
+    afterEach(function() {
+      jq.name_ = undefined;
+      delete jq.name_;
+    });
+
+    it('should return undefined when jq is not set, no jquery found (the default)', function() {
+      expect(jq()).toBe(undefined);
+    });
+
+    it('should return empty string when jq is enabled manually via [ng-jq] with empty string', function() {
```

Comment:
It's to differentiate between an `ng-jq` attribute without value that means "just use `jqLite` and not setting it at all which means "use the native detection mechanism".

Not the prettiest APIs of all, I agree, but it's used just in one place really so I didn't picked at it.